### ISSUE TITLE
False positive rule LOCAL_VARIABLE_EARLY_DECLARATION

### ISSUE THREAD

**Issue by Cheshiriks** [cqfn/diKTat#934]
The Diktat is triggered on LOCAL_VARIABLE_EARLY_DECLARATION in lambda

```
    override fun boo() {

        val listTestResult: MutableList<TestResult> = mutableListOf()
        
        files.chunked(warnPluginConfig.batchSize ?: 1).map { chunk ->
            handleTestFile(chunk.map { it.single() }, warnPluginConfig, generalConfig)
        }.forEach { listTestResult.addAll(it) }

    }

```
